services/dy-3dvis/simcoreparaviewweb/src/input-retriever.py

```diff
@@ -8,6 +8,8 @@
 import shutil
 from pathlib import Path
 
+import zipfile
+
 _LOGGER = logging.getLogger(__name__)
 
 print("Content-Type: text/html;charset=utf-8")
```

Comment:
What are these prints about? use logs?

Reply:
Those prints are needed for CGI scripting

Reply:
mmhhh ... does not feel the right way to do this. Does it need to be done upon import? can we perhaps encapsulate in a function that take a stream and defaults to stdout? This way we at least document what that does ... otherwise it can be easily mistaken by a quick log

Reply:
added some comments and the [link](https://docs.python.org/3/library/cgi.html)